src/App.tsx

```diff
@@ -1,10 +1,29 @@
 import React from "react";
+import { Quote } from "./components/atoms/Quote";
+import * as quoteData from "./utils/testdata.json";
 
 function App() {
+  const qoutes = [
+    quoteData.quote1,
+    quoteData.quote2,
+    quoteData.quote3,
+    quoteData.quote4,
+    quoteData.quote5,
+  ];
   return (
-    <div>
-      <p> Hello, world!</p>
-      <p>Hello, Krydderhyllen!</p>
```

Comment:
Bit rude that you removed this, but I understand

Reply:
Yeah, this was a really difficult choice to make :cry: